README.md

```diff
@@ -114,3 +113,21 @@ This will create the local database and insert some seeded data which are define
 | :----------------------------------------------------------: | :----------------------------------------------------------: | :----------------------------------------------------------: | :----------------------------------------------------------: | :----------------------------------------------------------: |
 |         [André Morad](https://github.com/AndreMorad)         |       [Nor Shiervani](https://github.com/norshiervani)       |         [Ahmad Yassin](https://github.com/akyassin)          |         [Micael Wollter](https://github.com/aohzork)         |          [Irvin Perez](https://github.com/Irvper77)          |
 
+
+
+## 4. Useful links
+
+- [Jira confluence - School project information/requirements (Requires authentication)](https://plushogskolan.atlassian.net/wiki/spaces/TO/pages/64061445/Systemkrav)
+
+- [Jira confluence - Project documentation (Requires authentication)](https://plushogskolan.atlassian.net/wiki/spaces/G5/overview)
+
+- [Jira software - Project board [Sprints, backlog, charts etc.] (Requires authentication)](https://plushogskolan.atlassian.net/secure/RapidBoard.jspa?rapidView=15&projectKey=G5)
+
+- [API](https://g5store.azurewebsites.net/) 
+
+- [Website](http://storefront5test.azurewebsites.net/)
+
+- [Website (Production)](https://storefront5prod.azurewebsites.net/)
+
+  
+
```

Comment:
Don't forget to add  production website.

[Website (Production)](https://storefront5prod.azurewebsites.net/)

Reply:
Fixed!